api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -153,4 +158,18 @@ private class PurchasesAPI {
         val customerInfo: CustomerInfo = logInResult.customerInfo
         LogInResult(customerInfo, created)
     }
+
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    fun checkAmazonConfiguration(context: Context, executorService: ExecutorService) {
```

Comment:
Is this for diagnostics? Maybe move that to a separate method? Otherwise if we accidentally make the whole Amazon configuration experimental we might not notice 🙈

Reply:
good catch